src/test/regress/expected/multi_foreign_key.out

```diff
@@ -920,9 +920,7 @@ SELECT create_reference_table('reference_table');
 (1 row)
 
 ALTER TABLE reference_table ADD CONSTRAINT fk FOREIGN KEY(referencing_column) REFERENCES referenced_local_table(id);
-ERROR:  referenced table "referenced_local_table" must be a distributed table or a reference table
-DETAIL:  To enforce foreign keys, the referencing and referenced rows need to be stored on the same node.
-HINT:  You could use SELECT create_reference_table('referenced_local_table') to replicate the referenced table to all nodes
+ERROR:  cannot create foreign key: could not find the coordinator node in metadata as it is not added as a worker
```

Comment:
"as it is node added as a worker" seems a little confusing

maybe omit that part and add an:
errhint("Use SELECT citus_set_coordinator_host(...) to add the coordinator to the metadata")

Reply:
omitting error part only for `as it is node added as a worker` would probably be a little hard, do you think should we hide error part in try/catch block in `table.c`, like that:

```
try:
    CreateCitusLocalTable(cascadeViaForeignKeys)
catch:
    ereport(err="cannot create foreign key", hint=capturedHint)
```

Reply:
well I think we should adjust the original error, but let's not do this now